basisu.vcxproj

```diff
@@ -87,8 +87,7 @@
       <Optimization>Disabled</Optimization>
       <SDLCheck>true</SDLCheck>
       <OpenMPSupport>true</OpenMPSupport>
-      <AdditionalIncludeDirectories>
-      </AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>.;transcoder;encoder;3rdparty\apg;3rdparty\jpeg-compressor;3rdparty\lodepng</AdditionalIncludeDirectories>
```

Comment:
@richgel999 
Do you still want to maintain the visual project files?
Since #173 these are not tested on appveyor anymore.
Also, the current cmake solution is superior.

Reply:
I already opened #174 😄

Reply:
Haha, get out of my head. You psych 😛

Reply:
#174 was declined so updating this seems like a good idea